src/components/lib/kernel/cos_kernel_api.c

```diff
@@ -938,18 +1001,37 @@ cos_hw_alloc(struct cos_compinfo *ci, u32_t bitmap)
 	return cap;
 }
 
+/* TODO: Can we alias/etc on this page with this logic? */
+void *
+cos_dcbpg_bump_allocn(struct cos_compinfo *ci, size_t sz)
```

Comment:
I don't understand these. We saw that the memory was allocated for dcb/scb from a kernel resource. Don' tunderstand this.